specs/agents/mobile/README.md

```diff
@@ -62,6 +62,7 @@ The following attributes do not have an OpenTelemetry semantic convention:
 | `telemetry.sdk.elastic_export_timestamp`| N/A: only relevant for APM server.     | `1658149487000000000` | :white_check_mark: yes | This is required to deal with the time skew on mobile devices. Set this to the timestamp (in nanoseconds) when the span is exported in the OpenTelemetry span processer. |
 | `type` | `transaction.type` | `mobile` :interrobang: | :white_check_mark: yes | :heavy_exclamation_mark: Need to define new values for transactions resulting from mobile interactions. |
 | `session.id`         | :heavy_exclamation_mark: not mapped yet         | `opbeans-swift`                  | :x: no | Some id for a session. This is not specified in OTel, yet. | 
+| `app.visibility`                         | :heavy_exclamation_mark: not mapped yet         | `foreground/background` | :x: yes                | This will be needed to tell whether the signal happened when the app was visible or in the background. |
```

Comment:
I'm open to other names, this is what I came up with trying to make it obvious but at the same time not to confuse it with app lifecycle states... Though not sure if it's a good name in general. @bryce-b @AlexanderWert (I'm tagging you because I think that's the only way to get notified on PR drafts)

Reply:
I think `visibility` is a good name

Reply:
Like this addition. Though I think we need to restructure the mobile spec slightly, since those are not pure span attributes anymore but will be added to other events as well. But we can do the restructuring in a separate PR